etc/templates/nginx.conf.template

```diff
@@ -39,7 +39,7 @@ http {
 
         location %(FORCE_SCRIPT_NAME)s {
             if (-f %(ROOT)s/var/maintenance.html) {
```

Comment:
Just a small comment, I never managed to get that maintenance working as this is handling 503 rather then 502 Bad Gateway. On the prod servers I have
```
    error_page 502  /500.html;
    
    location ~* ^/(50*\.html)$ {
        root /usr/local/etc/nginx/error;
    }
```
We should command to admin handle it in a global config